src/phoenix/server/api/dataloaders/latency_ms_quantile.py

```diff
@@ -113,11 +129,14 @@ async def _get_results(
     segment: Segment,
     params: Mapping[Param, list[ResultPosition]],
 ) -> AsyncIterator[tuple[ResultPosition, QuantileValue]]:
-    kind, (start_time, end_time), filter_condition = segment
+    kind, (start_time, end_time), filter_condition, session_filter_condition = segment
     stmt = select(models.Trace.project_rowid)
     if kind == "trace":
         latency_column = cast(FloatCol, models.Trace.latency_ms)
         time_column = models.Trace.start_time
+        if filter_condition:
+            sf = SpanFilter(filter_condition)
+            stmt = sf(stmt.join(models.Span)).distinct()
```

Comment:
i think you may have to do it this way for the quantiles to be correct
```suggestion
            stmt = stmt.where(models.Trace.id.in_(sf(select(models.Span.trace_rowid).distinct()).scalar_subquery()))
```